docs/modules/reference/pages/ingress.adoc

```diff
@@ -6,10 +6,10 @@ Ingress is not required, although it is a good thing to have.
 //external dependencies says it is.
 You could modify the Helm Chart to avoid Ingress altogether (or make it optional) and expose the WebUIs via `LoadBalancer` or `NodePort` services, but that won't be covered here.
 
-For example, when deploying the Helm Chart names `acme` (remember about the rules for the `namespace`) with a value of `k8s.agalue.net` for the `domain`, it would create an Ingress instance exposing the following resources via custom FQDNs:
+For example, when deploying the Helm Chart named `acme` (remember about the rules for the `namespace`) with a value of `k8s.agalue.net` for the `domain`, it would create an Ingress instance exposing the following resources via custom FQDNs:
```

Comment:
```suggestion
For example, when deploying the Helm chart named `acme` (remember about the rules for the `namespace`) with a value of `k8s.agalue.net` for the `domain`, it would create an Ingress instance exposing the following resources via custom FQDNs:
```

Should we include a link to the section on the namespace rules, or is it linkable?
Also, I'm not sure were consistent with capitalization of "charts" in our first draft. Kubernetes seems to capitalize it when it's referring to it as a "product", but when providing docs on configuration, they use lower. We should probably do a sweep and update ours too.

Reply:
Probably want to standardize on lower case.